### ISSUE TITLE
Spread syntax for dependencies

### ISSUE THREAD

**Issue by vaaralav** [msrose/babel-plugin-transform-amd-to-commonjs#57]
The plugin doesn't handle spread syntax for function arguments in definition.

The following snippet is not transformed correctly.

```js
define(['a', 'b', 'c'], function(...callbacks) {
  return (x) => callbacks.forEach((cb) => cb(x));
})
```

That is transformed to something which crashes with `Uncaught SyntaxError: Unexpected token ...`
```js
var ...callbacks = require('a');
    ^^^
require('b');
require('c');
```

The correct outcome would be something like this.

```js


module.exports = function() {
  var [...callbacks] = [require('a'), require('b'), require('c')];

  return (x) => callbacks.forEach((cb) => cb(x));
}();
```

**Comment by vaaralav** [comment]
Wow that's great! You fixed this really quick. I appreciate that 👏👏👏

**Comment by msrose** [comment]
Addressed by #58 and available in [v1.3.0](https://www.npmjs.com/package/babel-plugin-transform-amd-to-commonjs/v/1.3.0).
